src/openfluid/ui/waresdev/WareBuildOptionsWidget.cpp

```diff
@@ -34,6 +34,7 @@
   @file WareBuildOptionsWidget.cpp
```

Comment:
Signals connections are unnecessary, rebuilt the options struct once it is needed in the GetOptions() method